# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/ipc/RemoteAccessibleBase.h
# Commit: 1a9901750064
# Full Hash: 1a9901750064de9fc00eaeb6094e639b035eb278
# Author: Eitan Isaacson <eitan@monotonous.org>
# Date: 2021-08-27 03:42:22
# Regressor Bug: 1727643
# File Overlap Count: 1
# Description:
#   Bug 1727643 - P3: Make numeric value methods abstract in Accessible and wire it thru. r=morgan
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D123697
# ==============================================================================

diff -r c5789cd1f9c7 -r 1a9901750064 accessible/ipc/RemoteAccessibleBase.h
--- a/accessible/ipc/RemoteAccessibleBase.h	Thu Aug 26 19:40:05 2021 +0000
+++ b/accessible/ipc/RemoteAccessibleBase.h	Thu Aug 26 19:40:05 2021 +0000
@@ -164,8 +164,15 @@
     return HasGenericType(eNumericValue);
   }
 
+  // Methods that potentially access a cache.
+
   virtual ENameValueFlag Name(nsString& aName) const override;
 
+  virtual double CurValue() const override;
+  virtual double MinValue() const override;
+  virtual double MaxValue() const override;
+  virtual double Step() const override;
+
   /**
    * Allow the platform to store a pointers worth of data on us.
    */